### DISCUSSION TITLE
Directus and Cloudinary

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15426] [Archive]
Hi

We have all our images by Cloudinary. Is it possible to show by Directus collection (for example "Articles") a thumbnails of Images with external URL? Or maybe is it possible to connect our Cloudinary account to Directus?

Best regards,
Tony

**Comment by azrikahar** [comment]
> Is it possible to show by Directus collection (for example "Articles") a thumbnails of Images with external URL? 

It should be possible to add external images in WYSIWYG editor as mentioned here: https://github.com/directus/directus/discussions/10889#discussioncomment-1920490, but if the use case is more so to showcase it in a table layout for example, it's not possible yet and was discussed in #12634. But recently there's a new PR #15042 for external images so that may help 👍

> Or maybe is it possible to connect our Cloudinary account to Directus?

It's not officially supported yet. There is an article by the community where they integrated Cloudinary as mentioned here: https://github.com/directus/directus/discussions/9839#discussioncomment-1650781, but I don't have personal experience with it so I'm not able to provide more elaboration on that front.

Apart from that, feel free to also check out relevant Cloudinary related discussions such as #14649 and #10254, or more generally assets/storages adapter discussion such as #7612 and #14742.
